Sources/inc/CosmicCORSIKA.hh

```diff
@@ -47,22 +46,24 @@ struct Config
   using Name = fhicl::Name;
   using Comment = fhicl::Comment;
   fhicl::Sequence<std::string> showerInputFiles{Name("fileNames"),Comment("List of CORSIKA binary output paths")};
-  fhicl::Atom<int> firstEventNumber{Name("firstEventNumber"), Comment("Reference point on the Y axis"), 0};
-  fhicl::Atom<int> firstSubRunNumber{Name("firstSubRunNumber"), Comment("Reference point on the Y axis"), 0};
-  fhicl::Atom<int> maxEvents{Name("maxEvents"), Comment("Reference point on the Y axis"), 0};
-  fhicl::Atom<unsigned int> runNumber{Name("runNumber"), Comment("Reference point on the Y axis"), 0};
-  fhicl::Atom<std::string> module_label{Name("module_label"), Comment("Reference point on the Y axis"), ""};
-  fhicl::Atom<std::string> module_type{Name("module_type"), Comment("Reference point on the Y axis"), ""};
+  fhicl::Atom<int> firstEventNumber{Name("firstEventNumber"), Comment("First event number"), 0};
+  fhicl::Atom<int> firstSubRunNumber{Name("firstSubRunNumber"), Comment("First subrun number"), 0};
+  fhicl::Atom<int> maxEvents{Name("maxEvents"), Comment("Max number of events"), 0};
+  fhicl::Atom<unsigned int> runNumber{Name("runNumber"), Comment("First run number"), 0};
+  fhicl::Atom<std::string> module_label{Name("module_label"), Comment("Art module label"), ""};
+  fhicl::Atom<std::string> module_type{Name("module_type"), Comment("Art module type"), ""};
   fhicl::Atom<bool> projectToTargetBox{Name("projectToTargetBox"), Comment("Store only events that cross the target box"), false};
-  fhicl::Atom<float> showerAreaExtension{Name("showerAreaExtension"), Comment("Reference point on the Y axis"), 10000};
+  fhicl::Atom<float> showerAreaExtension{Name("showerAreaExtension"), Comment("Extension of the generation box on the xz plane"), 10000};
   fhicl::Atom<float> tOffset{Name("tOffset"), Comment("Time offset"), 0};
+  fhicl::Atom<float> lowE{Name("lowE"), Comment("lowE"), 1.3};
+  fhicl::Atom<float> highE{Name("highE"), Comment("highE"), 1e6};
   fhicl::Atom<float> fluxConstant{Name("fluxConstant"), Comment("Flux constant"), 1.8e4};
-  fhicl::Atom<float> targetBoxXmin{Name("targetBoxXmin"), Comment("Extension of the generation plane"), -5000};
-  fhicl::Atom<float> targetBoxXmax{Name("targetBoxXmax"), Comment("Extension of the generation plane"), 5000};
-  fhicl::Atom<float> targetBoxYmin{Name("targetBoxYmin"), Comment("Extension of the generation plane"), -5000};
-  fhicl::Atom<float> targetBoxYmax{Name("targetBoxYmax"), Comment("Extension of the generation plane"), 5000};
-  fhicl::Atom<float> targetBoxZmin{Name("targetBoxZmin"), Comment("Extension of the generation plane"), -5000};
-  fhicl::Atom<float> targetBoxZmax{Name("targetBoxZmax"), Comment("Extension of the generation plane"), 5000};
```

Comment:
Why don't you use the same definition of target Box as in defaultCRYconfig?

Reply:
These are the default values, but they definition in the FCL file is the same as defaultCRYconfig. However, to avoid confusion, I changed also the values here. Also, the comment is wrong and has been fixed.